### PR TITLE
fix: Missing ApplyOptions decorator

### PR DESCRIPTION
**Pull request by beyazit**
:yum:

### CODE REVIEW

**Comment by Nytelife26** [comment]
Is there a reason this needs `ApplyOptions`? It looks like it has not been used after import in your PR, either.

File: src/templates/typescript.ts [hunk 1]
```diff
@@ -50,13 +50,14 @@ export const template: Template = {
...
 									general.add('ping.ts', [
+										"import { ApplyOptions } from '@sapphire/decorators';",
 										"import { Command, CommandOptions } from '@sapphire/framework';",
 										'',
 										'@ApplyOptions<CommandOptions>({',
 										"	aliases: ['pong']",
 										'})',
 										'export class UserCommand extends Command {',
-										'	public async run(message: Message, args: Command.Args) {',
+										'	public async run(message, args: Command.Args) {',
```
Comment (Favna) [inline src/templates/typescript.ts:60]: Preettyyy sure this needs to be reverted?

**Review by Favna — CHANGES_REQUESTED** [review]

**Comment by Favna** [comment]
@Nytelife26 it is used
https://github.com/sapphiredev/create-sapphire-bot/blob/bcc6777fb8fce51a0fc72ab97e4ec7844df2b031/src/templates/typescript.ts#L55-L57

File: src/templates/typescript.ts (hunk 1 above)
Comment (beyazit) [reply src/templates/typescript.ts]: Sorry, I have specified the type of message object.

**Review by Favna — APPROVED** [review]

**Comment by Favna** [comment]
@all-contributors please add @beyazit for code

**Pull request merged**